pages/documentation/developer/repositories/powsybl-open-loadflow.md

```diff
@@ -7,6 +7,7 @@ The PowSyBl Open Load Flow [repository](https://github.com/powsybl/powsybl-open-
 ## Features
 
 - a load flow solver
+- a sensitivity analysis provider
```

Comment:
I think provider is too technical here. I would change to:
```
- power flow computations in both AC and DC mode
- sensitivity analysis
```

I would also add links to the relevant documentation pages:
https://www.powsybl.org/pages/documentation/simulation/powerflow/
https://www.powsybl.org/pages/documentation/simulation/sensitivity/